lib/Importer.js

```diff
@@ -401,6 +401,11 @@ export default class Importer {
       return undefined;
     }
 
+    const distanceToCurrentFile = (importPath: string): number => {
+      const match = path.relative(this.pathToCurrentFile, importPath).match(/\//g);
```

Comment:
The `path.relative` here is most often redundant, right? Unless `useRelativePaths` is `false`.